examples/packaging-with-repo/package-repository.yml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: packaging.carvel.dev/v1alpha1
+kind: PackageRepository
+metadata:
+  name: e2e-repo.test.carvel.dev
```

Comment:
Maybe a more user friendly name since this is an example? simple-packagerepository?